here/traffic/sql/dynamic_bins/create-table-congestion_raw_corridors.sql

```diff
@@ -0,0 +1,80 @@
+-- Table: gwolofs.congestion_raw_corridors
+
+-- DROP TABLE IF EXISTS gwolofs.congestion_raw_corridors;
+
+CREATE TABLE IF NOT EXISTS gwolofs.congestion_raw_corridors
+(
+    corridor_id smallint,
+    time_grp timerange NOT NULL,
+    bin_range tsrange NOT NULL,
+    tt numeric,
+    num_obs integer,
+    uri_string text COLLATE pg_catalog."default",
+    dt date,
+    hr timestamp without time zone,
```

Comment:
Since we have date as it's own field already, I might suggest letting this be a smallint. Otherwise, the current datetime `hr` field could serve both purposes, hour and date.